src/licensedcode/frequent_tokens.py

```diff
@@ -27,7 +27,7 @@
 from __future__ import print_function
 
 import itertools
-import string
+
```

Comment:
This commit is repetition of https://github.com/nexB/scancode-toolkit/pull/1534/commits/7b7946be65f09057a3e60d7731250c555b01b404 .